src/System.cpp

```diff
@@ -954,6 +954,7 @@ void System::begin() {
 #ifdef DS_CAP_MDNS
 #ifdef DS_CAP_SYS_LOG
   log->printf(TIMED("Starting mDNS service... "));
+  log->printf(TIMED("Hostname is:",hostname));
```

Comment:
The idea is right, but this would not compile, as TIMED() macro takes exactly one parameter. The second parameters needs to be taken out of brackets, like this:

```
log->printf(TIMED("Hostname is: %s"), hostname);
```

Reply:
I have changed the line of code. Thanks for correcting. I was not able to compile this code on my system due to "Android.h" missing.

Reply:
Thanks. This is the code for ESP8266 microcontroller. To compile it, you need to install Arduino IDE (https://www.arduino.cc/en/software) and then, in it, support for ESP8266 boards. https://github.com/esp8266/Arduino. All this stuff is not Android-related.